### DISCUSSION TITLE
osu!mania decimal scroll speed adjustment option

### DISCUSSION THREAD

**Discussion by DreamVFXs** [ppy/osu#30663] [Ideas] [answered]
there are times when i play on 26 scroll speed and it feels a little too slow, going up to 27 feels way to fast.
i wish there was an option to get something in between maybe 26.5 or even increments of 0.1 in the settings if possible.
![Screenshot 2024-08-25 042613211](https://github.com/user-attachments/assets/1e2ad45d-06ff-42bc-81e1-6049a1cf4efd)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/30832
